app/views/notifiers/course/consolidated_opening_reminder_notifier/opening_reminder/course_notifications/email.html.slim

```diff
@@ -0,0 +1,13 @@
+- course = @object
+- items_hash = Course::LessonPlan::Item.upcoming_items_from_course_by_type(course)
+/ Lesson plan item start at times could have been changed between the time the mailer job was
+/ enqueued and the time this view partial is called.
+/ Raise an exception if there are no longer any items so the consolidated email doesn't get sent.
+- raise 'No opening lesson plan items' if items_hash.empty?
+
+- message.subject = t('.subject', course: format_inline_text(course.title))
+p = t('.message')
+
+- items_hash.sort.each do |actable_type, items|
+  = render partial: actable_type_partial_path(@notification, actable_type),
+    locals: { items: items.sort_by! { |item| [item.start_at, item.title] }, course: course }
```

Comment:
I think we can do the sort on `start_at` and `title` on the first call in the class method of `Course::LessonPlan::Item`. Does the order get mixed up during this manipulation?